spec/factories/category_practices.rb

```diff
@@ -1,6 +1,9 @@
 FactoryBot.define do
   factory :category_practice do
     association :category
-    association :innovable, factory: :practice
+
+    factory :category_practice_for_product do
+      association :innovable, factory: :product
+    end
```

Comment:
to keep things copasetic with the factories I think this would be better off as a trait:

```
trait :for_product do
	association :innovable, factory: :product
end

trait :for_practice do
	association :innovable, factory: :practice
end
```